gpt_researcher/retrievers/searchapi/searchapi.py

```
@@ -27,7 +27,7 @@ def get_api_key(self):
         """
         try:
             api_key = os.environ["SEARCHAPI_API_KEY"]
-        except:
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)